### PR TITLE
CDM-17: New Divider component

### PR DESCRIPTION
**Pull request by catarinapm**
Line with full width (horizontal) and height (vertical).
Give an orientaion (horizontal or vertical)  and a color (category).

[WEB]
image

[MOBILE]
image

Reviewers:
- [ ] @mariasbl 
- [x] @danielaprado

### CODE REVIEW

**Review by danielaprado — APPROVED** [review]
LGTM 🚀

File: fe/app/components/Divider/Divider.tsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+import React from "react";
+
+const Divider = ({ category, orientation = "horizontal" }: DividerProps) => {
+  return orientation === "horizontal" ? (
+    <hr className={clsx("w-full", "border-" + category)} />
+  ) : (
+    <div
+      className={clsx(
+        "w-[1px] self-stretch",
+        category ? "bg-" + category : "bg-white"
```
Comment (mariasbl) [inline fe/app/components/Divider/Divider.tsx:12]: Just a suggestion, here´s another way of doing the className without clsx 
className=`w-[1px] self-stretch bg-${category ?? "white"}`

File: fe/app/components/Divider/Divider.tsx [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import { DividerProps } from "@/app/components/Divider/Divider.models";
+import clsx from "clsx";
+import React from "react";
+
+const Divider = ({ category, orientation = "horizontal" }: DividerProps) => {
+  return orientation === "horizontal" ? (
+    <hr className={clsx("w-full", "border-" + category)} />
```
Comment (mariasbl) [inline fe/app/components/Divider/Divider.tsx:7]: Another suggestion
className={`w-full border-${category}`}

**Review by mariasbl — CHANGES_REQUESTED** [review]

File: fe/app/components/Divider/Divider.tsx (hunk 2 above)
Comment (catarinapm) [reply fe/app/components/Divider/Divider.tsx:7]: thanks! updated

File: fe/app/components/Divider/Divider.tsx (hunk 1 above)
Comment (catarinapm) [reply fe/app/components/Divider/Divider.tsx:12]: thanks! updated

**Review by mariasbl — APPROVED** [review]

**Pull request merged**